runelite-client/src/main/java/net/runelite/client/plugins/teamcapes/TeamCapesOverlay.java

```diff
@@ -26,19 +26,32 @@
 
 import java.awt.Dimension;
 import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
 import java.util.Map;
 import javax.inject.Inject;
+import net.runelite.client.game.ItemManager;
 import net.runelite.client.ui.overlay.Overlay;
 import net.runelite.client.ui.overlay.OverlayPosition;
 import net.runelite.client.ui.overlay.OverlayPriority;
-import net.runelite.client.ui.overlay.components.LineComponent;
+import net.runelite.client.ui.overlay.components.ImageComponent;
 import net.runelite.client.ui.overlay.components.PanelComponent;
+import net.runelite.client.ui.overlay.OverlayManager;
+import net.runelite.client.ui.overlay.infobox.InfoBoxManager;
 
 public class TeamCapesOverlay extends Overlay
 {
 	private final TeamCapesPlugin plugin;
 	private final TeamCapesConfig config;
-	private final PanelComponent panelComponent = new PanelComponent();
+	private PanelComponent verticalContainer = new PanelComponent();
+	private PanelComponent horizontalContainer = new PanelComponent();
```

Comment:
Move this below injects and add empty line above it, also make it final and rename it t just panel